lib/graphql/transaction.ts

```diff
@@ -55,9 +70,22 @@ export class Transaction extends IterableModel<TransactionEntry> {
   async fetch(args?: FetchOptions): Promise<ResultPage<TransactionEntry>> {
     const result: Query = await this.client.rawQuery(FETCH_TRANSACTIONS, args);
 
-    const transactions = (result?.viewer?.mainAccount?.transactions?.edges ?? []).map((edge: TransactionsConnectionEdge) => edge.node);
+    const transactions = (
+      result?.viewer?.mainAccount?.transactions?.edges ?? []
+    ).map((edge: TransactionsConnectionEdge) => edge.node);
 
-    const pageInfo = result?.viewer?.mainAccount?.transactions?.pageInfo ?? { hasNextPage: false, hasPreviousPage: false};
+    const pageInfo = result?.viewer?.mainAccount?.transactions?.pageInfo ?? {
+      hasNextPage: false,
+      hasPreviousPage: false
+    };
     return new ResultPage(this, transactions, pageInfo);
   }
+
+  subscribe(handler: (event: SubscriptionEvent) => any): Unsubscribe {
```

Comment:
So this is what we will officially expose and document I think.

it has a "observable" style to it in that we subscribe passing a handler and it returns the method to unsubscribe, hope it looks fine? from a user's perspective it seemed the most simple approach to me